src/Sulu/Bundle/SnippetBundle/Tests/Functional/Controller/SnippetControllerTest.php

```diff
@@ -55,10 +55,9 @@ class SnippetControllerTest extends SuluTestCase
 
     public function setUp(): void
     {
-        parent::setUp();
```

Comment:
Can we just remove that?

Reply:
yes they is no parent implementation I removed it to be consistent